server/controllers/UserController.js

```diff
@@ -299,9 +296,22 @@ class UserController {
       const { email } = req.body;
       const foundUser = await User.findOne({ where: { email } });
 
+      if (!foundUser) {
```

Comment:
Is this `if(!foundUser) {}` statement necessary? If the user doesn't exist yet that means the new user can then sign up with the data not give 404 error

Reply:
Thanks for this comment, rafmme. Actually, this check is necessary because I need to make sure this user is in the database record or not. This is because only a user whose record has been previously created should be able to get a new verification email.